GLOSSARY.md

```diff
@@ -18,6 +18,10 @@ This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other su
 - ## Drift
 
     When a system's actual state has moved or is in the process of moving away from the [desired state](#desired-state), this is often referred to as drift.
+    
+- ## Pull
+
+    In contrast to traditional, CI/CD pipelines which "push" event based triggers to discover state changes, in GitOps desired state is "pulled". This state may be reconciled with local or remote actual states. In GitOps agents are expected to pull both desired and actual states without the requirement of events. Webhooks may be used to speed up this process, but should a webhook fail, the agent is still expected to discover changes to desired state by pulling from the source of truth. 
```

Comment:
How about moving some of the language we already have under the "Reconciliation" glossary item? IMO this part fits better under "Pull" - we should remove this part:

> Contrary to traditional CI/CD where automation is generally driven by pre-set triggers, in GitOps reconciliation is triggered whenever there is a divergence. Divergence could be due to the actual state unintentionally [drifting](https://github.com/open-gitops/documents/blob/release-v1.0.0/GLOSSARY.md#drift) from the desired state declarations, or a new desired state declaration version having been changed intentionally.

And add to something that explains this clearly within the context of Principle 3, like:

```suggestion
    [Principle 3](./PRINCIPLES.md) specifies the desired state must be "pulled" rather than "pushed", primarily because the software agents must be able to access the [desired state](#desired-state) from the [state store](#state-store) at _any_ time, not only when there is an intentional change in the state store triggering a push event.
    This is a prerequisite for [reconciliation](#reconciliation) to happen [continuously](#continuous), as specified in [principle 4](./PRINCIPLES.md).
    Note that – in contrast to traditional CI/CD, where automation is generally driven by pre-set triggers – in GitOps, [reconciliation](#reconciliation) is triggered _whenever_ there is a divergence.
    Divergence could be due to the actual state unintentionally [drifting](#drift) from the desired state declarations – not only due to a new desired state declaration version having been changed intentionally.
```